src/hooks/useColorGenerator/utils.test.ts

```diff
@@ -0,0 +1,24 @@
+import {getHue} from './utils';
+
+describe('getHue', () => {
+    test('returns values within the range [0, 360)', () => {
+        const MIN_HASH = -Math.pow(2, 31);
+        const MAX_HASH = Math.pow(2, 31);
+
+        for (let i = 0; i < 1000; i++) {
+            const hash = Math.random() * (MAX_HASH - MIN_HASH) + MIN_HASH;
```

Comment:
`(MAX_HASH - MIN_HASH) + MIN_HASH` is always equal to `MAX_HASH ` so this line could be written as:
```js
const hash = Math.random() * MAX_HASH;
```